### ISSUE TITLE
setHotbarSlotIndex($index, $slot) in Survival

### ISSUE THREAD

**Issue by deotern** [iTXTech/Genisys#1390]
#### Issue description

Before I update my Gensys to the latest, I'm using an old Genisys Build (about last week build), and I tried to use setHotbarSlotIndex at that time, it's working.
But last night, after I updated my Genisys to latest build, setHotbarSlotIndex isn't working anymore, but no crashes or any errors echoed from the console
And I tracked down Genisys commits, I found this commit d1db7b6cf2f011eb0df65da7ea384721ce9deaf1 , I think this commit is the most likely to cuase this bug

#### Steps to reproduce the issue

Basically, in my plugin does:
Step 1) Clear a player inventory
Step 2) $inv->addItem($ITEM);
Step 3) $inv->setHotbarSlotIndex(1, 0);
These is working before I updated Genisys, so DONT ever think my plugin bug... (I dont even changed any code related to set inventory in my plugin in these days)

#### OS and versions

* Genisys: Latest
* PHP: 7
* OS: Linux

#### Crashdump, backtrace, memory dumps, plugins list or other files
```
Paste here
```

**Comment by svilex** [comment]
@deotern You forgot $inv->sendContents()

**Comment by svilex** [comment]
setHotbarSlotIndex() won't send nothing you should use sendContents() to send the new slot index

**Comment by deotern** [comment]
Hmmm, okay
I'll try to adding a sendContents() and see if its working

But this is really strange, i didnt use sendContents before and its working xD :P idk why tho
